This file is part of MXE.
See index.html for further information.

Contains ad hoc patches for cross building.

[master 1ae0f2e] new version
 2 files changed, 11 insertions(+), 9 deletions(-)
From 1ae0f2e805de5bee3fa36584a87a47fe11893e8f Mon Sep 17 00:00:00 2001
From: MXE
Date: Sun, 17 Jun 2012 17:04:04 +0200
Subject: [PATCH] new version


diff --git a/configure.in b/configure.in
index e0c52a8..e4691a3 100644
--- a/configure.in
+++ b/configure.in
@@ -15,6 +15,16 @@ AC_CONFIG_HEADERS([config.h])
 
 AM_INIT_AUTOMAKE([1.9 no-dist-gzip dist-bzip2])
 
+# pkg-config check put earlier in configure.in, because before LIBGLADE did not 
+# use the cross-compiled-pkg-cfg.
+PKG_PROG_PKG_CONFIG
+
+PKG_CHECK_MODULES(LIBGLADE, [dnl
+  libxml-2.0 >= required_libxml_version dnl
+  atk >= required_atk_version dnl
+  gtk+-2.0 >= required_gtk_version dnl
+  glib-2.0 >= required_glib_version])
+
 GNOME_COMMON_INIT
 
 if test "$enable_debug" != "no"; then
@@ -40,14 +50,6 @@ AC_CHECK_FUNC(gtk_plug_get_type,
   AC_DEFINE(HAVE_GTK_PLUG,1,[gtk has GtkPlug/GtkSocket implementation]))
 LIBS=$save_LIBS
 
-PKG_PROG_PKG_CONFIG
-
-PKG_CHECK_MODULES(LIBGLADE, [dnl
-  libxml-2.0 >= required_libxml_version dnl
-  atk >= required_atk_version dnl
-  gtk+-2.0 >= required_gtk_version dnl
-  glib-2.0 >= required_glib_version])
-
 AC_MSG_CHECKING([for native Win32])
 case "$host" in
   *-*-mingw*)
diff --git a/glade/Makefile.am b/glade/Makefile.am
index f6271f7..0641232 100644
--- a/glade/Makefile.am
+++ b/glade/Makefile.am
@@ -13,7 +13,7 @@ if OS_WIN32
 export_symbols = -export-symbols glade.def
 
 install-libtool-import-lib:
-	$(INSTALL) .libs/libglade-2.0.dll.a $(DESTDIR)$(libdir)
+#	$(INSTALL) .libs/libglade-2.0.dll.a $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libglade-2.0.dll.a
-- 
1.7.9.5

